.circleci/config.yml

```diff
@@ -140,6 +140,36 @@ jobs:
       - run: composer functional
       - run: composer integration
 
+
+  # PHP 7.4 test with HIGHEST dependencies:
+  #   Determines whether a newer version of a dependency has broken Drush.
+  test_74_drupal91:
+    <<: *defaults
+    docker:
+      - image: wodby/php:7.4-dev
+        environment:
+          - MYSQL_HOST=127.0.0.1
+          - UNISH_DB_URL=mysql://root:@127.0.0.1
+      - image: circleci/mysql:5.7.24
+    steps:
+      - checkout
+      - run: sudo composer self-update --preview
+      - run: cp .docker/zz-php.ini /usr/local/etc/php/conf.d/
+      # We test with 9.0.0 so that \Unish\SecurityUpdatesTest can find an insecure drupal/core.
+      # Remove that assertion once testing with 9.0.0 becomes stale.
+      - run: composer require --dev drupal/core-recommended:~9.1.0 --no-update
+      # Replace alinks with semver for D9.
+      - run: composer remove --dev drupal/alinks --no-update
+      - run: composer require --dev drupal/semver_example:2.2.0 --no-update
+      - run: composer config platform.php 7.3.0
+      - run: composer update --no-scripts
```

Comment:
I think this line was added in https://github.com/drush-ops/drush/commit/e1f904cc49a676ad5240e83989d3dd86c7f26e20. Its not needed anymore in the 9.0 and 9.1 sections, I think?

Reply:
We need the `composer update`, but we can remove `--no-scripts` now. composer-test-scenarios were in use for PHP 5 testing back when that was added.

Reply:
Oh right. Its the following `composer install` that's superfluous.

Reply:
The `composer install` was necessary after `composer update --no-scripts`, to run the scripts. I'll take it out along with the --no-scripts, because it is superfluous once that's gone.